openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,21 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
```

Comment:
Why we changed the default value from False to None ? I think we should leave this field as it.